### DISCUSSION TITLE
EPUB 文件是怎么判断 code 的呢？

### DISCUSSION THREAD

**Discussion by DxPoly** [immersive-translate/immersive-translate#2231] [Help]
### 概述 | Summary

你好，请问一下，在翻译 epub 的时候，是怎么判断 code 的呢？
我发现 Pragmatic Bookshelf 的epub 格式电子书，code 部分都会被翻译。不确定是不是他们的 epub 文件格式问题，想跟你们确认一下这块是怎么判断的，看看能不能解决，或者至少可以确认了问题之后去给他们发邮件问问。
类似截图这样的:
![91730101545_ pic](https://github.com/user-attachments/assets/9e7d84d8-c47d-40aa-8829-7d6ec26f43aa)

**Comment by keyboard3** [comment]
这个需要打开网页开发者模式，查看这部分代码区域是否和文字区域的元素样式有区别。不过从你图上表现看到，该epub文件并没有对这部分代码内容做区分，只是作为普通文本元素。这种情况就无法避免被翻译
